apps/server/src/modules/registration/api/dto/response/registration-item.response.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsEnum } from 'class-validator';
+import { LanguageType } from '@shared/domain/interface';
+import { Consent } from '../../../domain/type';
+
+export class RegistrationItemResponse {
+	@ApiProperty()
+	public id: string;
+
+	@ApiProperty()
+	public email: string;
+
+	@ApiProperty()
+	public firstName: string;
+
+	@ApiProperty()
+	public lastName: string;
+
+	// do we want to expose the password in the response?
+	@ApiProperty()
+	public password: string;
+
+	@ApiProperty({ enum: Consent, isArray: true, enumName: 'Consent' })
+	@IsArray()
+	@IsEnum(Consent, { each: true })
+	public consent: Consent[];
+
+	@ApiProperty({ enum: LanguageType, enumName: 'LanguageType' })
+	@IsEnum(LanguageType)
+	public language: LanguageType;
+
+	@ApiProperty()
+	@IsArray()
+	public roomIds: string[];
+
+	@ApiProperty()
+	public registrationHash: string;
```

Comment:
definetly dont expose the registrationHash, since we now decided for that to be the sole authenticating factor for the real user.